plugins/email_service/app/controllers/email_service/configsets_controller.rb

```diff
@@ -69,19 +83,34 @@ def destroy
       end
       redirect_to plugin('email_service').configsets_path and return
       rescue Elektron::Errors::ApiResponse => e
-        flash[:error] = "Status Code: #{e.code} : Error: #{e.message}"
+        error = "#{I18n.t('email_service.errors.configset_delete_error')} #{e.message}"
+        Rails.logger.error error
+        flash[:error] = error
       rescue Exception => e
-        flash[:error] = "Status Code: 500 : Error: #{e}"
-      
+        error = "#{I18n.t('email_service.errors.configset_delete_error')} #{e.message}"
+        Rails.logger.error error
+        flash[:error] = error
+
     end
 
 
     def edit
```

Comment:
Do yo use the edit action?

Reply:
Does this ```before_action :set_configset, only: %i[edit, destroy, update]``` need an explicit edit action defined in controller or not ? If not needed, I can remove the edit action.

Reply:
removed.